.github/workflows/release.yml

```diff
@@ -61,28 +61,28 @@ jobs:
           DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
 
       - name: 🛠️ Run Build
-        run: cargo build -p pg_cli --release --target ${{ matrix.config.target }}
+        run: cargo build -p pglt_cli --release --target ${{ matrix.config.target }}
 
       # windows is a special snowflake to, it saves binaries as .exe
       - name: 👦 Name the Binary
         if: matrix.config.os == 'windows-latest'
         run: |
           mkdir dist
-          cp target/${{ matrix.config.target }}/release/pg_cli.exe ./dist/pgcli_${{ matrix.config.target }}
+          cp target/${{ matrix.config.target }}/release/pglt.exe ./dist/pglt_${{ matrix.config.target }}
```

Comment:
```suggestion
          cp target/${{ matrix.config.target }}/release/pglt_cli.exe ./dist/pglt_${{ matrix.config.target }}
```

Reply:
are you sure? I think with the new config in the toml it writes it as `pglt`

![Screenshot 2025-02-12 at 09 30 09](https://github.com/user-attachments/assets/562683dc-650a-429f-bca6-fb5d7d6b3161)

Reply:
Ah my mistake, I used the `main` branch without the binary config. Go ahead! ❤️